### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by Meta-Maxim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# World Portals
+
+A module to add teleportation portals. Portals currently occupy a block and transport any entities that enter the occupied block to the portals destination world position.
```
Comment (vampcat) [inline README.md]: "A module to add teleportation portals."
This feels.. abrupt. Rephrasing it to "This module adds ..." makes it feel more natural.

"... to the portals destination ..."
You missed an apostrophe here. (Not trying to be a grammar nazi, the lack of apostrophe changes the meaning, throwing off the reader)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Future Features
```
Comment (vampcat) [inline README.md]: Unless there is good reason, I would push this down to the very end.
(keep in mind that these are ToDos, that may or may not ever get implemented. This readme is for the players as much as is it for the developers, and both of them are concerned primarily with what the module *currently* offers, not what it *might* offer some day)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- Block-linked portals that remove when the block is removed.
+- Cross world portals
+
+## PortalComponent
```
Comment (vampcat) [inline README.md]: Make this a subheading, and shift it down into a `Implementation` heading.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+A portal component represents a one way portal and has a block coordinate location and a world position destination.
+
+## PortalRasterizer & related.
```
Comment (vampcat) [inline README.md]: "PortalRasterizer & related."
Change the & to and. Further, and remove the period at the end of the heading. Further, "& related." : related what? It ends way too abruptly.
Note that this heading should also become a subheading under `Implementation`.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+An abstract class that generates rectangular portal structures in the world, to be implemented in modules that need quick portals in the world or as a usage demo.
```
Comment (vampcat) [inline README.md]: I don't see any reference to any sort of "related" stuff here.

`or as a usage demo`
That is not something you explicitly need to mention.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+An example portal structure used in Underworld module:
```
Comment (vampcat) [inline README.md]: Make a separate heading for this : `Sample Implementation` or `Module Usage` or whatever suits your fancy.

**Review by vampcat — CHANGES_REQUESTED** [review]
Apart from the comments, I would like to see a `Getting Started` or `How to use this Module` section, *aimed at players*. Assume I just downloaded Terasology off the internet and downloaded this module, how do I spawn my own portals and play around with them? (if it's not possible, then mentioned it as such - `This module is aimed as a depency for other modules, and cannot be used directly by the player` or something)

**Comment by Meta-Maxim** [comment]
@vampcat Fixed and resubmitted.

**Review by vampcat — APPROVED** [review]
I'm satisfied with it.

**Comment by casals** [comment]
Merging as per conversation history/current README version

**Pull request merged**